package.json

```
@@ -1,61 +1,63 @@
 {
-  "name": "@smg-automotive/example",
+  "name": "@smg-automotive/eslint-config",
   "version": "0.0.0-development",
   "description": "A boilerplate package setup",
-  "main": "dist/cjs/index.js",
-  "module": "dist/esm/index.js",
-  "files": [
-    "dist/**/*"
-  ],
   "publishConfig": {
     "access": "public"
   },
   "scripts": {
     "version": "npm run build",
-    "build": "rimraf dist && rollup -c",
-    "lint": "eslint --ext ts,js,tsx,jsx,json .",
+    "build": "rimraf dist && tsc -p tsconfig.build.json --outDir dist",
+    "lint": "npm run build && eslint --ext ts,js,tsx,jsx,json ./src --ignore-path ./.eslintignore.internal",
     "format": "npm run lint -- --fix",
     "typecheck": "tsc --noEmit",
-    "test": "jest",
-    "test:debug": "node --inspect-brk --inspect=127.0.0.1:9229 ./node_modules/jest/bin/jest.js --runInBand"
+    "test": "npm run build && node dist/__tests__/test.js"
   },
   "author": "SMG Automotive",
   "license": "MIT",
   "repository": {
     "type": "git",
-    "url": "https://github.com/smg-automotive/fe-example-pkg.git"
+    "url": "https://github.com/smg-automotive/eslint-config-pkg.git"
   },
-  "homepage": "https://github.com/smg-automotive/fe-example-pkg#readme",
-  "devDependencies": {
-    "@rollup/plugin-commonjs": "22.0.0",
-    "@rollup/plugin-node-resolve": "13.3.0",
-    "@rollup/plugin-typescript": "8.3.2",
-    "@testing-library/jest-dom": "5.16.4",
-    "@testing-library/react": "13.2.0",
-    "@testing-library/user-event": "14.1.1",
-    "@types/jest": "27.5.0",
+  "homepage": "https://github.com/smg-automotive/eslint-config-pkg#readme",
+  "dependencies": {
+    "@next/eslint-plugin-next": "^12.1.6",
+    "@rushstack/eslint-patch": "1.1.3",
     "@typescript-eslint/eslint-plugin": "5.22.0",
+    "@typescript-eslint/eslint-plugin-tslint": "5.22.0",
     "@typescript-eslint/parser": "5.22.0",
     "eslint": "8.14.0",
-    "eslint-config-airbnb": "19.0.4",
+    "eslint-config-next": "^12.1.6",
```

Renovate wants dependencies to be pinned:
```suggestion
    "eslint-config-next": "12.1.6",
```